sql/core/src/test/scala/org/apache/spark/sql/StatisticsCollectionSuite.scala

```diff
@@ -168,6 +157,60 @@ class StatisticsCollectionSuite extends StatisticsCollectionTestBase with Shared
       assert(stats.simpleString == expectedString)
     }
   }
+
+  test("change stats after truncate command") {
```

Comment:
I use "change" instead of "invalidate" in the test name, because in the near future when we support auto update, we can add check logic in the same test cases.